operator/src/main/java/org/bf2/operator/managers/IngressControllerManager.java

```diff
@@ -158,6 +159,14 @@ public class IngressControllerManager {
     Quantity maxIngressThroughput;
     @ConfigProperty(name = "ingresscontroller.max-ingress-connections")
     int maxIngressConnections;
+    @ConfigProperty(name = "ingresscontroller.hard-stop-after")
+    Optional<String> hardStopAfter;
```

Comment:
Since these appear in the application.properties, will they ever actually be optional?  I'm fine leaving the code this way, just more of a general question.